docs/6.bridge/3.bridge-composition-api.md

```diff
@@ -94,7 +94,9 @@ Remove the `defineNuxtPlugin` wrapper:
 ```diff
 - import { defineNuxtPlugin } from '@nuxtjs/composition-api'
 - export default defineNuxtPlugin((ctx, inject) => {})
-+ export default (ctx, inject) => {}
++ export default defineNuxtPlugin(nuxtApp => {
```

Comment:
It's possible to provide an explanation similar to Nuxt 3.
https://nuxt.com/docs/migration/plugins-and-middleware#plugins

```ts
export default defineNuxtPlugin(nuxtApp => {
  // now available on `nuxtApp.$injected`
  nuxtApp.provide('injected', () => 'my injected function')

  // You can alternatively use this format, which comes with automatic type support
  return {
    provide: {
      injected: () => 'my injected function'
    }
  }
})
```